build.gradle

```diff
@@ -0,0 +1,26 @@
+
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.4.31'
+    id 'java-gradle-plugin'
+    id 'idea'
```

Comment:
I believe it would be a good idea to add this to other Gradle builds in appland, it just sits there until somebody actually wants to use IntelliJ and generates with this the proper config files. @dustinbyrne